### ISSUE TITLE
Posibility for importing js modules

### ISSUE THREAD

**Issue by discoverr** [wmonk/create-react-app-typescript#73]
I am currently using this awesome package in order to have typescript without sacrificing the benefits of create-react-app. 

Even though i would love to refactor all of our code to TypeScript we dont have the ability to do so. Instead we were trying to make declaretion files for the modules we kept as es6. But when we tried to import the es6 module the code was not transpiled. 

Could we somehow allow the package to keep the standard babel config for .js files :-) ?

**Comment by bryanmig** [comment]
There is another issue open (#60) for converting to use `at-loader` instead and that has built in support for using Babel.  Before I learned about this CRA fork, I was using it successfully in an ejected CRA with a setup that sounds like what youre looking for.

See description at https://github.com/s-panferov/awesome-typescript-loader

**Comment by FredrikNoren** [comment]
@bryanmig will this also fix jest test js importing? I think I'm running into this issue right now, when requiring a `.js` file with a `.d.ts` declaration in jest test I just get the name of file back. I.e. `import * as X from './x' X is now just `x.js`.

**Comment by bryanmig** [comment]
I'm sorry - I dont recall having that same problem so I'm not sure if it will be resolved.

**Comment by ceylons** [comment]
Even though that createing a custom build config it might be a nice implementation in create-react-app-typescript :-) ?

**Comment by rluiten** [comment]
I have a generated peg.js parser I want to use in a parser.js file.
I imported it into one of my components and it appears to import fine and lets me run it in my browser in dev mode fine it seems at moment.

I then added a test spec for it so I could setup a few test cases for the paraser and find that no matter what I do cant get more the name of the module "parser.js" as the result of `import parser = require('./parser.js');`

Is there anything I can do that might let me get tests going ?

Its been a while since I've worked much with typescript and I am still quite new to CRA so not sure I can contribute much to actually work on create-react-app-typescript itself to help address the behavior I am seeing.
